bin/mdsctl

```diff
@@ -429,19 +429,19 @@ installIstio() {
 
     installNamespace istio-system
     pause 2
-    helm template ${configs[istio]}/install/kubernetes/helm/istio-init \
-      --name istio-init --namespace istio-system | \
-      kubectl apply -f - ${configs[dryrun]:+--dry-run}
+    helm install ${configs[istio]}/install/kubernetes/helm/istio-init \
+      --name istio-init --namespace istio-system \
+      ${configs[dryrun]:+--dry-run}
```

Comment:
Can you explain this change to me? I'm not sure I'm following it.

Reply:
@wesleyfsmith definitely. thx for inquiring. there are a couple of ways to apply changes to a k8s cluster, classically via the `kubectl apply` command or using a higher-order utility like that of `helm`. in this change-set we actually used both, helm for materializing (transform a collection of deployment data with variable overrides to a serialized/deployable form) the deployment data and piping the results on into kubectl. for lacuna/mds fully leveraging helm, when possible, is desired. helm additionally provides deployment lifecycle operations but only when the entire lifecycle is managed via helm ... thus this pr.